keep-ui/app/dark-mode-toggle.tsx

```diff
@@ -43,7 +43,7 @@ export default function DarkModeToggle() {
 
   return (
     <button
-      className="flex items-center justify-between space-x-3 w-full text-sm p-1 text-gray-700 hover:bg-gray-200 font-medium rounded-lg hover:text-orange-500 focus:ring focus:ring-orange-300"
+      className="flex items-center justify-between space-x-3 w-full text-sm p-1 text-gray-700 hover:bg-stone-200/50 font-medium rounded-lg hover:text-orange-400 focus:ring focus:ring-orange-300"
```

Comment:
The toggle itself is now "different" from the rest of the items, which looks kinda weird -- maybe "adjust" it to match the new style?

Reply:
Resolved with https://github.com/keephq/keep/pull/1011/commits/9ca2fce012e87371f08c3ee64ad692c5a3a2e494